cmd/operator-sdk/generate/bundle/bundle.go

```diff
@@ -17,14 +17,19 @@ package bundle
 import (
 	"errors"
 	"fmt"
+	"io/ioutil"
 	"os"
 	"path/filepath"
+	"sort"
 	"strings"
 
 	"github.com/operator-framework/operator-registry/pkg/lib/bundle"
+	yaml "gopkg.in/yaml.v3"
```

Comment:
could we please use `sigs.k8s.io/yaml` instead of this one for we still closer as possible to rip all these other modules to dealing with yaml as well?

Reply:
That isn't possible until we bump operator-registry versions since the struct being marshalled only has `yaml` tags. I don't think we'll bump versions until after v1.0.